tests/mastra/tools/sql-tool.test.ts

```
@@ -0,0 +1,629 @@
+    }) as any;
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+
+    mockExecute.mockResolvedValue(createMockResult());
+    mockRunMigrations.mockResolvedValue();
+
+    vi.spyOn(console, 'error').mockImplementation(() => {});
+
+    process.env = { ...originalEnv };
+    delete process.env.DATABASE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)